src/app/app.js

```diff
@@ -1,12 +1,35 @@
-import React from 'react';
-
-function App() {
-  return (
-    <div>
-      Hello world!
-    </div>
-  );
+import React, {Component} from 'react';
+import { connect } from 'react-redux';
+
+import { getPhotos } from '../actions/index.js';
+import Image from '../components/image.js';
+
+const mapStateToProps = state => ({
```

Comment:
Totally optional, but you can use destructuring here, e.g.
`const mapStateToProps = { photos } => ({ photos });`